tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -2126,6 +2162,120 @@ TEST_P(OpConverter_FP32_Test, ConvertTranspose) {
   }
 }
 
+TEST_P(OpConverter_FP32_Test, ConvertTile) {
+  Scope s = Scope::NewRootScope();
+  auto input = ops::Placeholder(s.WithOpName("input"), tf_type_);
+  auto weights = ops::Placeholder(s.WithOpName("weights"), DT_INT32);
+  auto tile = ops::Tile(s.WithOpName("my_tile"), input, weights);
+  const NodeDef& node_def = tile.operation.node()->def();
+
+  struct TileParam {
+    std::vector<int> input_dims;
+    std::vector<int> multiplier;
+    std::vector<float> tensor;
+    // Concrete (static) output dimensions, including batch size as first dim
+    std::vector<int> expected_output_dims;
+    std::vector<int> expected_results;
+    // Expected status of conversion (with concrete error message)
+    Status status;
+    // Expected status of BuildAndRun
+    Status runtime_status;
+  };
+
+  std::vector<TileParam> test_params = {
+      TileParam{{1, 2, 3},  // input dims
+                {1, -2, 1}, // multiplier
+                {},
+                {},         // output dims
```

Comment:
s/output dims/expected_output_dims/

Use correct field names for these comments. Would it hurt to label them all? For the next person to debug.

Reply:
Done